packages/flutter_test/lib/src/binding.dart

```diff
@@ -507,6 +507,12 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   }
 
   Future<Null> _runTestBody(Future<Null> testBody(), VoidCallback invariantTester) async {
+    // Delay this function by a microtask.
+    // Otherwise it will open as cope immediately, which then open the
+    // `asyncBarrier` is invoked. The `asyncBarrier` is immediately following
+    // the call to `testZone.runBinary(_runTestBody`, so delaying by one
```

Comment:
This is an older version of your comment, with a missing right parenthesis and unparseable text at the beginning

Reply:
Thanks.
I somehow got my versions mixed up...